src/Sulu/Bundle/WebsiteBundle/DependencyInjection/SuluWebsiteExtension.php

```diff
@@ -120,6 +121,10 @@ public function load(array $configs, ContainerBuilder $container)
 
             // default local provider
             $container->setAlias('sulu_website.default_locale.provider', $config['default_locale']['provider_service_id']);
+
+            // add alias for default controller
+            $container->setAlias(DefaultController::class, 'sulu_website.default_controller')
```

Comment:
aliases can not be tagged so we need to define it in php inside an if statement

Reply:
What exactly are you saying here? Isn't it possible to set the `public="true"` attribute on the alias as it was before?

Reply:
Defining the alias in the `xml` fails on symfony 5 in the `admin context` as the target service `sulu_website.default_controller` does not exist there.

Reply:
and `aliases` can not be tagged with `<tag name="sulu.context" context="website" />`